pkgs/tools/security/step-ca/0.15.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, buildGoPackage, fetchFromGitHub }:
+let
+  sha256 = "01y4wi6j1frfpd6ndwrxw4sr9rwhj8m96mvp76rnszzwgvlydgd4";
+  vendorSha256 = "1is4rvga3anmzqqd1bmdw3014lxrvnm19jrfpyvmdqh7pc7cm4i5";
+  version = "0.15.5";
+in
```

Comment:
This `let in` isn't useful
You can do
```nix
buildGoModule rec {
```
and still use version in `src.rev` and `buildPhase` without the `let in`

Reply:
Looking all over the file for places to update hashes and version tag is rather inconvenient. Unless it's code style requirement for nixpkgs, I'd rather keep it this way.

Reply:
It's not a requirement, but it's never done like that, and it's not a big inconvenient

Reply:
I'm not sure it's a good thing, since it just adds code for nothing. The repo is large enough.
It could also break some scripts that updates packages automatically.
If you use a good text editor, you can just search for `version` or `sha256` and update things quickly.
I would advise you to follow the same style as the other packages and get rid of that `let in`.

Reply:
My argument is that this way data is clearly separated from the process, this makes release tracking easy to automate. «Not done before» (everything was «not done before» at some point in time), appealing to the text editor capabilities (if we all use a good text editor, it would also wrap long `meta.description` for us neatly), repository size (what percentage we're talking about, precisely?) aren't good arguments. Besides, if you look at `pkgs/applications/networking/cluster/nomad`, you will see exactly the same approach.

Reply:
Ok then, I won't prevent you from doing what you want, but at least use `inherit`.

Reply:
Done!